### ISSUE TITLE
Hide "Open Attachment" and/or  "Copy URL" when using an attachment

### ISSUE THREAD

**Issue by allkindsofralph** [binwiederhier/ntfy#1242]
It would be nice if we could remove/hide these buttons to remove distractions from other action buttons we may have added.

**Comment by jwhumphrey** [comment]
I'm all for it! There are others I wish to be optional as well. I'm currently using "attach" for images but I have no need for the 'Open' and 'Browse' that are displayed on the notification. I just need to pic, not to interact with it.

**Comment by luckycold** [comment]
Yeah, I have a requester that prevents me from using as many actions as I'd like. I'm just using attachments as a thumbnail for the request.

**Comment by binwiederhier** [comment]
Thank you for your suggestion and for using ntfy. I don't think I want to do this right now, so I respectfully will close this issue.
